examples/local/mtp_to_ovito.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+MTP_PREDICTION=./experiments/mtp_find_region/predictions.csv
```

Comment:
This file doesn't exist. Should it be `data/mtp_templates/mtp_predictions.csv`? Why not move that file here?

Reply:
yes, moved to examples/